src/routes/index.tsx

```diff
@@ -85,8 +149,9 @@ export const PAGES = [
 ]
 
 const RoutesList = () => {
-  const RedirectToDashboard = () => <Navigate to={PAGES[0].path} replace />
-  const routes = PAGES.filter((r) => r.element)
+  const pages = getPages()
```

Comment:
Must be called outside this function, because it's a hook

Reply:
> Must be called outside this function, because it's a hook

If you move usePages outside of a react component, you get this warning: 
index.tsx:29 Warning: Invalid hook call. Hooks can only be called inside of the body of a function component. 

I think I misunderstood you or something on this one:)

Reply:
My bad, it should be inside the component,  you're right ✅